pkg/etwlogrus/hook.go

```diff
@@ -1,35 +1,73 @@
+//go:build windows
 // +build windows
 
 package etwlogrus
 
 import (
+	"errors"
 	"sort"
 
-	"github.com/Microsoft/go-winio/pkg/etw"
 	"github.com/sirupsen/logrus"
+
+	"github.com/Microsoft/go-winio/pkg/etw"
 )
 
+const DefaultEventName = "LogrusEntry"
```

Comment:
Do we have to export this?

Reply:
I didnt know if an end user would want it for code, otherwise we are setting it without them knowing what it is.

Reply:
I think it's fine to leave unexported. They can just override it if they want to.